### ISSUE TITLE
FPS Unlocked bugged for roblox xbox players

### ISSUE THREAD

**Issue by CrazyKebab** [axstin/rbxfpsunlocker#1783]
**So basically, without the fps unlocker, my xbox controller works perfectly fine. But when I put on the fps unlocker, it is really hard to move my camera around. This is extremely annoying as I play on an xbox controller. Please fix this, I would really appreciate it!****

**Comment by axstin** [comment]
Does this happen when you set the cap to 60 or lower?

**Comment by CrazyKebab** [comment]
No. It works perfectly fine when I set the cap to 60 or lower. But when I set it any higher, the camera doesn’t work properly.

**Comment by axstin** [comment]
This sounds like an issue with the Roblox code that handles controller input. Does this happen on every game? Do you use an Xbox 360 or Xbox One controller?

**Comment by CrazyKebab** [comment]
I only use FPS unlocker when I play roblox. I use an Xbox one controller.

**Comment by CrazyKebab** [comment]
(repost)
This happens on every roblox game I play. I use an xbox one controller.

**Comment by axstin** [comment]
If possible, could you record and upload a video of moving the camera at 60fps vs higher framerates? I believe this is something I can't fix and has to do with Roblox's camera/controller CoreScript code.
